src/main/java/hudson/plugins/sonar/action/SonarCacheAction.java

```diff
@@ -89,7 +90,7 @@ ProjectInformation get(SQProjectResolver resolver, long lastBuildTime, SonarAnal
   }
 
   @VisibleForTesting
-  static boolean isEntryValid(@Nullable ProjectInformation cached, long lastBuild) {
+  static boolean isEntryValid(ProjectInformation cached, long lastBuild) {
     if (cached == null) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2583)